docs/fuel-101/sequencer.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Sequencer
+category: Fuel 101
+---
+
+# What is a Sequencer?
+
+> A sequencer is a component utilized in Layer 2 scaling solutions, such as Optimistic Rollups and certain types of Zero-Knowledge Rollups to order transactions.
+
+Since rollups strategically leverage the consensus and data availability of another layer 1 chain, they depend on a sequencer for transaction ordering.
+
+By efficiently ordering transactions, sequencers allow Layer 2 solutions to significantly reduce transaction processing times and costs. This efficiency gain is essential for scaling blockchain networks and improving their usability for a wider range of applications, making the trade-off an integral part of optimizing blockchain performance.
+
+## Shared Sequencer
+
+> A shared sequencer model involves multiple entities or nodes taking turns or sharing the responsibility of sequencing transactions.
+
+Fuel leverages a shared sequencer an Application Blockchain Interface (ABCI), integrated with the Cosmos network. Validators are responsible for
```

Comment:
I think this sentence needs a touch up from a grammar perspective.

Also, is it true it's integrated with the Cosmos network? I'm not sure that's accurate.

Reply:
Yes I confirmed this with @DefiCake

Reply:
Changed to "Fuel leverages a shared sequencer on the Cosmos network. Validators on this network are responsible for..."